pkg/apply/destroyer.go

```diff
@@ -75,6 +79,7 @@ func (d *Destroyer) Run(inv inventory.InventoryInfo) <-chan event.Event {
 		err := d.PruneOptions.Prune(inv, nil, sets.NewString(), taskContext, prune.Options{
 			DryRunStrategy:    d.DryRunStrategy,
 			PropagationPolicy: metav1.DeletePropagationBackground,
+			InventoryPolicy:   option.InventoryPolicy,
```

Comment:
Should there be unit tests with different inventory policies for `Destroyer` now?

Reply:
We'll address this in future PRs.